src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -41,28 +44,44 @@ class CallActor extends Actor {
 
   private def creator: ActorRef = _creator.get
 
-  override def receive: Receive =
+  private def notStarted: Receive = {
     LoggingReceive {
-      case Start(call, symbolStore) =>
+      case Start(call, backend, symbolStore) =>
         val inputs = symbolStore.locallyQualifiedInputs(call)
         val tryCommandLine = call.task.command.instantiate(inputs)
         tryCommandLine match {
           case Success(commandLine) =>
             _creator = Option(sender())
-            self ! Run(commandLine)
+            self ! Run(commandLine, backend, call, symbolStore)
             creator ! Started
-          case Failure(t) =>
-            sender ! Failed(t)
+            context.become(started)
```

Comment:
It would be interesting to compare/contrast, if you're looking for weekend projects... :smile: